models/topics.model.js

```diff
@@ -1,7 +1,7 @@
 const db = require("../db/connection");
 
 exports.readAllTopics = () => {
-  console.log("hello from the model");
+  //console.log("hello from the model");
```

Comment:
console.log's/comments that are not needed should be removed before pushing your code up to github